backend/director/db/__init__.py

```
@@ -2,17 +2,28 @@
+from .postgres.db import PostgresDB
 
 db_types = {
     DBType.SQLITE: SQLiteDB,
+    DBType.POSTGRES: PostgresDB,
 }
 
-
-def load_db(db_type: DBType) -> BaseDB:
+def load_db(db_type: DBType = None) -> BaseDB:
+    if db_type is None:
+        db_type = os.getenv("DB_TYPE", DBType.SQLITE)
```

Ruff (0.8.2)

Undefined name `os`

(F821)